src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -2540,63 +2163,56 @@ void WriteTabularLifeCycleCostReport(EnergyPlusData &state)
         columnHead(4) = "Present Value";
         columnHead(5) = "Present Value Factor";
         totalPV = 0.0;
-        rowHead(numRows + 1) = "TOTAL";
-        for (jObj = 1; jObj <= (elcc->numRecurringCosts + elcc->numNonrecurringCost + elcc->numResourcesUsed); ++jObj) {
-            offset = countOfCostCat;
-            rowHead(jObj) = elcc->CashFlow(offset + jObj).name;
-            {
-                auto const SELECT_CASE_var(elcc->CashFlow(offset + jObj).Category);
-                if (SELECT_CASE_var == costCatMaintenance) {
-                    tableBody(1, jObj) = "Maintenance";
-                } else if (SELECT_CASE_var == costCatRepair) {
-                    tableBody(1, jObj) = "Repair";
-                } else if (SELECT_CASE_var == costCatOperation) {
-                    tableBody(1, jObj) = "Operation";
-                } else if (SELECT_CASE_var == costCatReplacement) {
-                    tableBody(1, jObj) = "Replacement";
-                } else if (SELECT_CASE_var == costCatMinorOverhaul) {
-                    tableBody(1, jObj) = "Minor Overhaul";
-                } else if (SELECT_CASE_var == costCatMajorOverhaul) {
-                    tableBody(1, jObj) = "Major Overhaul";
-                } else if (SELECT_CASE_var == costCatOtherOperational) {
-                    tableBody(1, jObj) = "Other Operational";
-                } else if (SELECT_CASE_var == costCatConstruction) {
-                    tableBody(1, jObj) = "Construction";
-                } else if (SELECT_CASE_var == costCatSalvage) {
-                    tableBody(1, jObj) = "Salvage";
-                } else if (SELECT_CASE_var == costCatOtherCapital) {
-                    tableBody(1, jObj) = "Other Capital";
-                } else if (SELECT_CASE_var == costCatWater) {
-                    tableBody(1, jObj) = "Water";
-                } else if (SELECT_CASE_var == costCatEnergy) {
-                    tableBody(1, jObj) = "Energy";
-                } else {
-                    tableBody(1, jObj) = "-";
-                }
+        rowHead(numRows + 1) = TotalUC;
+        for (jObj = 0; jObj < (elcc->numRecurringCosts + elcc->numNonrecurringCost + elcc->numResourcesUsed); ++jObj) {
+            offset = CostCategory::Num;
+            rowHead(jObj + 1) = elcc->CashFlow[offset + jObj].name;
+            switch (elcc->CashFlow[offset + jObj].Category) {
+            case (CostCategory::Maintenance):
```

Comment:
No need to put parentheses around the case labels.  No need to hold it up, but in the future don't add them.

Reply:
Got it. Happened because I memorized this pattern. Will not add them in the future. 
![image](https://user-images.githubusercontent.com/45446967/142459733-1474dab1-9b7f-4142-adfc-008c33c48284.png)